Makefile

```diff
@@ -37,13 +37,14 @@ ifeq ($(shell uname), Darwin)
 MAKE := make
 endif
 
-kkm-pkg: ## Build KKM module self-extracting package.
+.kkm-pkg:
+	makeself -q kkm ${BLDTOP}/kkm.run "beta-release" ./installer/build-script.sh
+
+kkm-pkg: .kkm-pkg ## Build KKM module self-extracting package.
 	@if ! type makeself >& /dev/null ; then echo Please install \"makeself\" first; false; fi
 	@# We only needs source code, so running 'clean' first
 	$(MAKE) MAKEFLAGS="$(MAKEFLAGS)" -C ${TOP}/kkm/kkm clean
```

Comment:
You've changed the order of the steps here.
Before we ran make and then makeself,
now we run makeself and then make.
Does the makeself depend on what was produced by the make invocation?

Reply:
good catch. However make does clean, so it is not producing anything, it is removing things. Still, need to restore the order